<ui version="4.0" >
 <class>UninstallFrm</class>
 <widget class="QDialog" name="UninstallFrm" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Uninstall Rockbox</string>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="6" row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="rbutilqt.qrc" >:/icons/wizard.jpg</pixmap>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Please select the Uninstallation Method</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Uninstallation Method</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QRadioButton" name="CompleteRadioBtn" >
        <property name="text" >
         <string>Complete Uninstallation</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="smartRadioButton" >
        <property name="text" >
         <string>Smart Uninstallation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Please select what you want to uninstall</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2" >
    <widget class="QGroupBox" name="smartGroupBox" >
     <property name="title" >
      <string>Installed Parts</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QListWidget" name="UninstalllistWidget" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>341</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>291</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>&amp;Uninstall</string>
       </property>
       <property name="icon" >
        <iconset resource="rbutilqt.qrc" >:/icons/go-next.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="rbutilqt.qrc" >:/icons/process-stop.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rbutilqt.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>UninstallFrm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>562</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel" >
     <x>500</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>UninstallFrm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>658</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel" >
     <x>611</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
